installer/scripts/service_control

```diff
@@ -145,8 +145,10 @@ setup_variables()
     fi
 
     WS_STATUS=0
-    check_omsadmin_conf $initial_conf_dir
-    WS_STATUS=$?
+    if [ $1 != "scom" ]; then
```

Comment:
Better practice to check that $1 is not in the `NON_OMS_WS` list (my main concern is that this same check needs to be made for the "LAD" workspace as well). Other than that, this looks good.